src/sentry/api/endpoints/team_projects.py

```diff
@@ -214,4 +217,16 @@ def post(self, request: Request, team) -> Response:
                 sender=self,
             )
 
+            # Create project option to turn on ML similarity feature for new EA projects
+            seer_eligible_platform = (
+                project.platform and project.platform in SEER_ELIGIBLE_PLATFORMS
```

Comment:
```suggestion
                project.platform in SEER_ELIGIBLE_PLATFORMS
```
nit: shouldn't need to check `project.platform` here unless I'm missing something

Reply:
good catches, thanks!